mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionDaoImpl.java

```diff
@@ -205,6 +206,19 @@ public Future<JobExecution> updateBlocking(String jobExecutionId, JobExecutionMu
     return promise.future();
   }
 
+  @Override
+  public Future<Boolean> deleteJobExecutionsByIds(List<String> ids, String tenantId) {
+    Promise<RowSet<Row>> promise = Promise.promise();
+    try {
+      String query = format(DELETE_BY_IDS_SQL, convertToPsqlStandard(tenantId), TABLE_NAME, ids.stream().collect(Collectors.joining("','")));
```

Comment:
Hi @SatishkumarPurswani Do we also need to clear job_execution_progress and job_execution_source_chunks tables to not keep "corrupted" data.

Reply:
@AivarIUsupov - Changes have been done as per our new approach now, Thanks 🙂